test/integration/helpers.bash

```diff
@@ -72,6 +73,7 @@ assert_package_and_deps_installed() {
 
     tdeps_file="/hab/pkgs/${ident}/TDEPS"
     if [ -e "${tdeps_file}" ]; then
+        # shellcheck disable=2013
```

Comment:
```
In ./test/integration/helpers.bash line 76:
        for dep in $(cat "${tdeps_file}"); do
                   ^-- SC2013: To read lines rather than words, pipe/redirect to a 'while read' loop.
```
This is probably safe since our TDEPS files *should* never have any whitespace characters in them, but to explicitly operate on a line-by-line basis, it's more robust to use:
```bash
        while IFS= read -r dep; do
            assert_package_installed "${dep}"
        done < "${tdeps_file}"
```
See https://github.com/koalaman/shellcheck/wiki/SC2013

Reply:
Makes sense